config/esp32/components/chip/Kconfig

```diff
@@ -494,6 +494,12 @@ menu "CHIP Device Layer"
                 The amount of time (in milliseconds) after which the CHIP platform will deactivate the soft-AP
                 if it has been idle.
 
+        config WIFI_AP_RANDOM_MAC
+            bool "Enable Random MAC address"
+            default n
```

Comment:
Shouldn't the default be "yes"?  The spec-required behavior is the "yes" behavior....

Reply:
Done